app/controllers/questions_controller.rb

```diff
@@ -1,7 +1,13 @@
 class QuestionsController < ApplicationController
-  before_action :logged_in_user, only: [:new, :create]
+  before_action :load_question, only: [:show, :edit, :update, :destroy]
+  before_action :correct_user, only: [:edit, :update, :destroy]
```

Comment:
why do you need to find `question` 2 times ?

Reply:
its a mistake, fixed already.